docs/dev/marketplace_offering.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Build A Marketplace Offering
+type: documentation
+aliases: 
+- 'developers/marketplace/'
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/marketplace_offering.md"
+further_reading:
+- link: "https://www.datadoghq.com/partner/"
+  tag: "Partner Network"
+  text: "Datadog Partner Network"
+- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
+  tag: "Blog"
+  text: "Expand your monitoring reach with the Datadog Marketplace"
+- link: "/developers/integrations/create_a_tile"
+  tag: "Documentation"
+  text: "Create a Tile"
+- link: "/developers/integrations/agent_integration"
+  tag: "Documentation"
+  text: "Create an Agent-based Integration"
+---
+
+## Overview
+
+The [Datadog Marketplace][2] is a digital marketplace where Technology Partners can list their paid offerings to Datadog users.  
+
+While the **Integrations** page includes integrations built by both Datadog and Technology Partners at no cost, the **Marketplace** page is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings, including Agent-based or API-based integrations, software licenses, and professional services.
+
+{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}
+
+## List an offering 
+
+The following types of offerings are supported on the Datadog Marketplace:
+
+Integrations
+: Marketplace integrations that submit third-party data to (or pull data from) a user's Datadog account through the [Datadog Agent][15] or the [Datadog API][16]. These integrations can contain a variety of data types, such as metrics, events, logs, traces, and more.
+
+Software licenses
+: Software licenses enable you to deliver and license software solutions to customers through the Datadog Marketplace.
+
+Professional services
+: Professional services enable you to offer your team's services for implementation, support, or management for a set period of time.
+
+## Join the Datadog Marketplace 
+
+Marketplace Partners have unique benefits that are not available to Technology Partners who list out-of-the-box integrations:
+ 
+- **Go-to-market collaboration** including a blog post, a quote for a press release, and social media amplification, with access to dedicated sales and marketing resources focused on accelerating partner growth. 
+- **Training and support** for sales enablement in the form of on-demand courses to quickly onboard sales and technical teams.
```

Comment:
This may confuse partners, who could expect to have dedicated training sessions with our sales team, which we are not able to provide. 

Separately, I think we should add reaching Datadog's 25,000+ customers in this benefits section. That number is public in our quarterly earnings report.

Reply:
Changed to "Training and support for internal sales enablement".

@alexvetras on the bit about reaching customers, I don't think thats a benefit that's unique to the marketplace page. Should we instead add that to the general benefits section in the readme.md (overview doc)?